UnityServer/functions/src/controller/shared.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright 2018 Google LLC. All Rights Reserved.
```

Comment:
probably good to update this to 2020 instead of 2018

Reply:
Yep, forgot to update the dates